doc/source/MANUAL.rst

```diff
@@ -138,58 +138,70 @@ filters will be run. For example, if you called ``stestr run foo bar`` this
 will only run the tests that have a regex match with foo **or** a regex match
 with bar.
 
-stestr allows you do to do simple test exclusion via passing a rejection/black
-regexp::
+stestr allows you do to do simple test exclusion via passing a rejection or
```

Comment:
The previous usage of "rejection" only really made sense to explain black-regex, which is just an indication of how bad a name it was:

```suggestion
stestr allows you do to do simple test exclusion via passing a
```